docs/Installation-Guide.rst

```diff
@@ -179,26 +224,25 @@ Then run the following commands:
 
 **Note**: glibc >= 2.14 is required.
 
-OSX
-^^^
-
-Install **gcc** and **Open MPI** first:
+macOS
+^^^^^
+See instruction above to install **gcc** and **CMake**
 
 .. code::
 
-  brew install openmpi
-  brew install cmake
-  brew install gcc --without-multilib
+   brew install openmpi
+   brew install cmake
+   brew install gcc --without-multilib
 
-Then run the following commands:
+Then, run the following commands to install LightGBM:
 
 .. code::
 
-  git clone --recursive https://github.com/Microsoft/LightGBM ; cd LightGBM
-  export CXX=g++-7 CC=gcc-7
-  mkdir build ; cd build
-  cmake -DUSE_MPI=ON ..
-  make -j4
+   git clone --recursive https://github.com/Microsoft/LightGBM ; cd LightGBM
+   export CC=/usr/local/bin/gcc CXX=/usr/local/bin/g++ CXX1X=/usr/local/bin/g++
+   mkdir build ; cd build
+   cmake -DUSE_MPI=ON ../CMakeLists.txt
```

Comment:
do we need `../CMakeLists.txt` specifically? (`cmake ..` should be enough)

Reply:
`cmake ..` doesn't work
cmake needs the path to the file on macOS